build/test.sh

```diff
@@ -78,7 +78,6 @@ check_dependencies() {
 check_dependencies
 
 echo "Running tests:"
-go test -v -installsuffix "static" ${TARGETS}
```

Comment:
Why is this removed?

Reply:
Originally we were builing tests with `go test -i` and then running it (next few lines), but then the flag was deprecated and removed from this line, but that meant that we run the tests twice, which doesn't make much sense.